%YAML:1.0

#--------------------------------------------------------------------------------------------
# Camera Parameters. Adjust them for your camera.
#--------------------------------------------------------------------------------------------
CameraMat: !!opencv-matrix
  rows: 3
  cols: 3
  dt: d
  data: [951.41298945, 0.0,      776.64295079,
         0.0,     951.14501547,  306.57177568,
         0.0,     0.0,      1.0     ]

DistCoeffs: !!opencv-matrix
  rows: 5
  cols: 1
  dt: d
  data: [-1.24217316e-01, 1.18406210e-01, -9.13914012e-05, -1.11054590e-04, -8.54989083e-03]

Camera.width: 1520
Camera.height: 568
